packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -40,12 +30,30 @@ export class TestFixture {
     return this;
   }
 
+  public withHash(hash: string) {
+    this.hash = hash;
+    return this;
+  }
+
+  public withElement(e: HTMLElement) {
+    this.searchInterface.append(e);
+    return this;
+  }
+
   public withAlias(aliasFn: TestFeature) {
     this.aliases.push(aliasFn);
     return this;
   }
 
   public init() {
+    cy.visit(buildTestUrl(this.hash)).injectAxe();
```

Comment:
Is there a performance impact of injecting Axe on every init?

Reply:
I don't believe so, i moved the visit call to init to allow changing the URL settings, it shouldn't impact performance. Cypress tests length is not significantly increased. What affect performance in a good way, is using before instead of beforeAll and making multiple rendering checks with the same setup :)